### DISCUSSION TITLE
Edit form is creating duplicate

### DISCUSSION THREAD

**Discussion by elliewalkersm** [nss-evening-cohort-14/evening-client-side#231] [Help Me!] [answered]
Ellie Walker

Issue description:
My edit board is creating a duplicate board instead of editing the selected board

What I have Googled:

I rewatched the class recording on update

What I have tried:

- Passing boardInfo into the edit form
- Passing user instead of uid

Links:
https://github.com/elliewalkersm/react-team-pinterest/blob/ejw-update-board/src/components/AddBoardForm.js
https://github.com/elliewalkersm/react-team-pinterest/blob/ejw-update-board/src/components/BoardCard.js
https://github.com/elliewalkersm/react-team-pinterest/blob/ejw-update-board/src/helpers/data/boardsData.js

**Comment by TrinityChristiana** [comment, marked as answer]
Looks like in your AddBoardForm you're [looking for `board.firebaseKey`](https://github.com/elliewalkersm/react-team-pinterest/blob/ejw-update-board/src/components/AddBoardForm.js#L34) when board's state object does not have `fireaseKey` key, [it does have a key of `id`](https://github.com/elliewalkersm/react-team-pinterest/blob/ejw-update-board/src/components/AddBoardForm.js#L20)

**Reply by elliewalkersm** [reply]
That was it! I forgot we named it id instead of firebaseKey. Thanks!
